shared/templates/static/ansible/disable_prelink.yml

```diff
@@ -0,0 +1,12 @@
+# platform = multi_platform_rhel,multi_platform_fedora
+# reboot = false
+# strategy = restrict
+# complexity = low
+# disruption = low
+- name: disable prelinking
+  lineinfile:
+    path: /etc/sysconfig/prelink
+    regexp: '^PRELINK'
```

Comment:
It is not a big deal, but it would be better to have regexp like `^PRELINKING=`

Reply:
Ok changed regex.